src/client/features/search/entity-summary-box.js

```diff
@@ -0,0 +1,161 @@
+const React = require('react');
+const h = require('react-hyperscript');
+const Link = require('react-router-dom').Link;
+const queryString = require('query-string');
+const _ = require('lodash');
+
+const config = require('../../../config');
+let { DATASOURCES } = require('../../../models/entity/summary');
+
+const ENTITY_OTHER_NAMES_LIMIT = 4;
+const ENTITY_DESCRIPTION_WORD_LIMIT = 40;
+const ENTITY_SUMMARY_DISPLAY_LIMIT = 6;
+
+//Temporary - to be dealt with in #1116 (https://github.com/PathwayCommons/app-ui/issues/1116)
+const DATASOURCE_NAMES = {
+  [DATASOURCES.NCBIGENE]: {
+    displayName: 'NCBI Gene',
+    linkUrl: DATASOURCES.NCBIGENE
+  },
+  [DATASOURCES.HGNC]: {
+    displayName: 'HGNC',
+    linkUrl: 'http://identifiers.org/hgnc.symbol/'
+  },
+  [DATASOURCES.UNIPROT]: {
+    displayName: 'UniProt',
+    linkUrl: DATASOURCES.UNIPROT
+  },
+  [DATASOURCES.GENECARDS]: {
+    displayName: 'GeneCards',
+    linkUrl: 'https://www.genecards.org/cgi-bin/carddisp.pl?gene='
+  },
+};
+
+class EntitySummaryBox extends React.Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      expanded: props.expanded || false,
+      descriptionExpanded: false
+    };
+
+  }
+  render(){
+    let { summary } = this.props;
+    let { dataSource, displayName, localID, description, aliasIds, xref } = summary;
+    // Retrieve the HGNC symbol (http://www.pathwaycommons.org/sifgraph/swagger-ui.html)
+    let hgncSymbol = summary.xref[ DATASOURCES.HGNC ] || localID; // Prefix is hgnc
+    let sortedLinks = _.toPairs( xref ).concat([[ dataSource, localID ]])
+        .sort( (p1, p2) => p1[0] > p2[0] ? 1: -1 )
```

Comment:
what do p1[0] and p2[0] represent here?

Reply:
Those are the datasource names. This code is temp and will go away as part of #1116.